yoti-sdk-spring-boot-example/src/main/java/com/yoti/api/examples/springboot/IdentityLoginController.java

```diff
@@ -73,6 +73,10 @@ public String receipt(@RequestParam("receiptId") String receiptId, Model model)
 
         if (receipt == null || receipt.getError().isPresent()) {
             model.addAttribute("error", receipt.getError().get());
+
+            receipt.getErrorReason()
+                    .ifPresent(reason -> model.addAttribute("errorReason", reason));
+
```

Comment:
How does this work?
`if (receipt == null ... )`
then  
`receipt.getError().get()`

that leads to a `NullPointerException`

Reply:
even if the receipt null comes from a very unlike exception this is ofc wrong. Fixing it.